napari/_qt/qt_viewer.py

```diff
@@ -1105,6 +1106,7 @@ def on_mouse_press(self, event):
         """
         self._process_mouse_event(mouse_press_callbacks, event)
 
+    @qthrottled(timeout=5)
```

Comment:
We do not have the guarantee that the QApplication instance is created before the import of this file. 

And creating an QObject (timer in decorator) before QApplication may lead to random crashes of application/tests

Reply:
I moved it in the canvas initialization.